Cargo.toml

```
@@ -4,7 +4,7 @@ version = "0.2.0"
 authors = ["Vaskel"]
 edition = "2024"
 rust-version = "1.85"
-description = "Soteria is a Polkit authentication agent written in GTK"
+description = "Soteria is a Polkit authentication agent written in Relm4"
```

```suggestion
description = "Soteria is a Polkit authentication agent written in GTK"
```

Still in GTK, relm4 still uses GTK behind the scenes.